.circleci/config.yml

```diff
@@ -387,18 +387,21 @@ jobs:
         description: The golang arch
         type: string
         default: amd64
-    executor: golang
+    executor: # we are using linux/arm64 and linux/amd64 vm's for build
+      name: vm-<< parameters.arch >>
+    environment:
+      GOPATH: /home/circleci/.go-kuma-go
```

Comment:
I think that's not needed anymore (exported in install_build_tools)

Reply:
Cleaned